src/containers/Favourite.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { useQuery } from 'react-query';
+import { useDispatch } from 'react-redux';
+import Housetile from '../components/Housetile';
+import { selectedHouseAction, setStatusAction } from '../actions';
+
+const url = 'https://floating-harbor-48342.herokuapp.com/favourites';
+
+const fetchHouses = () => fetch(url, {
```

Comment:
It's a better practice (and in fact, the proper practice) to perform your fetching in another file. Components/containers aren't responsible for making API requests.